Binary files /dev/null and b/node_modules/expo-modules-core/android/build/tmp/kotlin-classes/debug/expo/modules/rncompatibility/ReactNativeFeatureFlags.class differ
diff --git a/node_modules/expo-modules-core/src/web/index.web.ts b/node_modules/expo-modules-core/src/web/index.web.ts
index 611564a..d9b46db 100644
--- a/node_modules/expo-modules-core/src/web/index.web.ts
+++ b/node_modules/expo-modules-core/src/web/index.web.ts
@@ -1,5 +1,5 @@
 import { EventEmitter, NativeModule, SharedObject, SharedRef } from './CoreModule';
-import uuid from '../uuid';
+import * as uuid from 'uuid';
 
 // jest-expo imports to this file directly without going through the global types
 // Exporting the types to let jest-expo to know the globalThis types
